### ISSUE TITLE
Featured Image missing alt text when cropped

### ISSUE THREAD

**Issue by jdelia** [ampproject/amp-wp#3094]
When adding a featured image to a post, if you select to not crop the image, the correct alt text is displayed in the HTML. If you select to crop the image, the alt text is empty. Tried this on Twentynineteen theme, ASTRA theme and Genesis Essence Pro theme. Same results across the board.

See the following screenshots from the ASTRA theme. 

<img width="1469" alt="Image 2019-08-26 at 12 13 59 PM" src="https://user-images.githubusercontent.com/3217274/63705910-26e8fc80-c7fc-11e9-974b-ca1d155adaf9.png">
<img width="484" alt="Image 2019-08-26 at 12 15 33 PM" src="https://user-images.githubusercontent.com/3217274/63705927-30726480-c7fc-11e9-9d31-708318485b2d.png">
<img width="1424" alt="Image 2019-08-26 at 12 16 19 PM" src="https://user-images.githubusercontent.com/3217274/63705940-39fbcc80-c7fc-11e9-8c9d-f5dd7e6f887c.png">

Environment:
WordPress 5.2.2
AMP Plugin 1.2.1
Gutenberg Plugin 6.3.0
